src/node_lkl.cc

```diff
@@ -150,3 +148,39 @@ NAN_METHOD(syscall) {
 		}
 	}
 }
+
+NAN_METHOD(parseDirent64) {
```

Comment:
This is not a library function.
The only API we expose is readdir and readdirSync.
This function is not supposed to be called by anything else than readdir or listdir from readdir.js.
One call to readdir or readdirSync in will often require multiple getdents64 and parseDirent64 calls.
Constructing multiple arrays on the c side and merging them in javascript will make the code longer and slower.